### ISSUE TITLE
Copyright Information Page

### ISSUE THREAD

**Issue by Nakaner** [rurseekatze/OpenRailwayMap#88]
We need a copyright information page like [OSM copyright page](https://www.openstreetmap.org/copyright) in addition to our map attribution at the lower right corner. This page should contain following information for data users:
- OpenStreetMap license and short information (with links to [OSM copyright page](https://www.openstreetmap.org/copyright) and OSM wiki)
- license of background map tiles and OpenRailwayMap map tiles (should be the same)

**Comment by grischard** [comment]
Yes please! We'd like to include it in the editor layer index, but can't:

https://github.com/osmlab/editor-layer-index/pull/720
